src/games/strategy/engine/data/Route.java

```diff
@@ -355,8 +355,8 @@ public String toString() {
    *         the route consists of only a starting territory
    */
   public Territory getEnd() {
-    if (m_steps.size() == 0) {
-      return null;
+    if (m_steps.isEmpty()) {
+      return m_start;
```

Comment:
Change was required given the test case, a comment would have been good though, yeah.